source/development/contributing.rst

```diff
@@ -136,6 +136,30 @@ development directly on it. Follow these steps:
     * if the **reviewers** are taking too long, try contacting the :term:`PR`
       assignee.
 
+.. _split_prs:
+
+Chain of PRs
+############
+
+A chain of PRs is a sequence of PRs that are dependent on each other. For
+example, a PR that introduces a new subsystem of a feature inherently depends
+the acceptance of the PR that introduces the feature itself. Therefore, the PR
```

Comment:
inherently depends on the PR that introduces the feature itself.

Reply:
Fixed.